include/circt/Dialect/FIRRTL/NLATable.h

```diff
@@ -49,6 +50,15 @@ class NLATable {
   /// Remove the NLA from the analysis.
   void erase(NonLocalAnchor nlaOp);
 
+  void commonNLAs(StringAttr mod1, StringAttr mod2,
```

Comment:
Some documentation on this would be great.

Reply:
Added the comments.